docs/detections/api/exceptions/api-create-exception-item.asciidoc

```diff
@@ -1,24 +1,24 @@
 [[exceptions-api-create-exception-item]]
 === Create exception item
```

Comment:
Might want to update this to: Create exceptions that apply to multiple rules

Reply:
Resolved in [3e3bea2](https://github.com/elastic/security-docs/pull/4021/commits/3e3bea2962d2a03df9672e65d5827f9db681e6a1).